hail/python/hail/methods/statgen.py

```diff
@@ -1810,13 +1810,12 @@ def pc_relate(call_expr, min_individual_maf, *, k=None, scores_expr=None,
     int_statistics = {'kin': 0, 'kin2': 1, 'kin20': 2, 'all': 3}[statistics]
 
     ht = Table._from_java(scala_object(Env.hail().methods, 'PCRelate')
-                          .apply(Env.hc()._jhc,
-                                 g._jbm,
-                                 scores_table._jt,
-                                 min_individual_maf,
-                                 block_size,
-                                 min_kinship,
-                                 int_statistics))
+                          .pyApply(g._jbm,
+                                   Env.spark_backend('pc_relate')._to_java_ir(scores_table.collect(_localize=False)._ir),
```

Comment:
so, here "spark_backend" is basically the leader node's JVM doing some light text munging? We're not actually computing the scores table, right?

Reply:
`Env.backend()` returns the current backend object.  `Env.spark_backend(op)` returns the backend if it is the Spark backend, and otherwise throws an error.  So operations that are only supported by the Spark backend should be prefixed with `spark_backend(op)`.

Reply:
Ah, got it.